test/acceptor_SUITE.erl

```diff
@@ -399,6 +403,42 @@ ssl_accept_error(_) ->
 	true = is_process_alive(AcceptorPid),
 	ok = ranch:stop_listener(Name).
 
+ssl_dedicated_sockets(_) ->
+	case do_os_supports_reuseport() of
+		true ->
```

Comment:
Just do a do_ssl_dedicated_sockets function instead of indenting like this. Call it if true.